python/setup.py

```diff
@@ -408,7 +409,7 @@ def onerror(function, path, excinfo):
             nonlocal result
             if excinfo[1].errno != errno.ENOENT:
                 msg = excinfo[1].strerror
-                logger.error("cannot remove {}: {}" % (path, msg))
+                logger.error("cannot remove %s: %s", path, msg)
```

Comment:
Please use `{}` (for the consistent style)

Reply:
OK.